pipelines/build/regeneration/build_pipeline_generator.groovy

```diff
@@ -1,32 +1,67 @@
+import java.nio.file.NoSuchFileException
+
 node('master') {
   def retiredVersions = [9, 10, 12, 13, 14]
+  def generatedPipelines = []
+
+  // Load gitUri and gitBranch. These determine where we will be pulling configs from.
+  def repoUri = "$REPOSITORY_URL" != "" ? REPOSITORY_URL : "https://github.com/AdoptOpenJDK/openjdk-build.git"
+  def repoBranch = "$REPOSITORY_BRANCH" != "" ? REPOSITORY_BRANCH : "master"
+  
```

Comment:
Replying to all the whitespace reviews here:

I'm a big fan of whitespaces as I feel it makes code easier to read and structure.

If everything was squashed down like the pipeline build base file used to be, it will take longer for newcomers to get to grips with the code (including myself when I first joined the project). Too much concatenation is naturally harder to read and debug. On the other hand, excessive white space can make files unbearably long at times but seeing as this is only one file, I feel justified with the existing whitespace changes in this file. 

I'm open to thoughts or opinions of any kind though 🙂

Reply:
Sorry Morgan I should have been more clear. I am fine with whitespace for formatting. I was pointing out instances where there were extra whitespace. I tend to catch these because my editor highlights them in red.
ex
![image](https://user-images.githubusercontent.com/6430052/98823526-36079a80-2400-11eb-83c7-582db74297ce.png)
You can see on lines 59,60,76,88,97 there is extra whitespace on empty lines or after the last char of a line.

Reply:
Ah ok no problem I can clear that up. Out of interest, what editor / plugin are you using that does that?

Reply:
Sublime. I think the package is TrailingSpaces but it could be Highlight Whitespaces.

Reply:
Done. I found an equivalent package for Vscode :)